tests/integration/requirements.yml

```diff
@@ -3,3 +3,7 @@ collections:
   - ansible.windows
   - ansible.utils # ipv6 filter
   - amazon.cloud # used by integration tests - rds_cluster_modify
+  - ansible.netcommon
+  - community.crypto
+  - community.aws
```

Comment:
For testing amazon.aws (and for testing community.aws, amazon.aws)  this **needs** to come directly from git, not from galaxy.  Otherwise we can't coordinate breaking changes between the two repos.

Reply:
@GomathiselviS `community.aws` should not be defined here, it should be kept in the required project on the integration job because we are testing both collections with the same branch (for instance `stable-X` on will be tested with `community.aws:Stable-X`)